src/middleware/express.ts

```diff
@@ -64,7 +65,11 @@ export async function makeMiddleware(
     logger.add(transport);
   }
 
-  const auth = transport.common.stackdriverLog.logging.auth;
+  const auth = (
+    transport.common.redirectToStdout
+      ? (transport.common.stackdriverLog as LogSync)
+      : (transport.common.stackdriverLog as Log)
+  ).logging.auth;
```

Comment:
nit: what auth we have in logSync? if logSync does not have auth functionality let's condition its use depending on the redirect flag.

Reply:
Auth is used to retrieve projectId and generate a [resource](https://github.com/googleapis/nodejs-logging/blob/7434d649a3ac0ecbf6299719403df35a2c1a833e/src/utils/metadata.ts#L187) - based on our samples, the LogSync is used like:
  ```js
  // Optional: Create and configure a client
  const logging = new Logging();
  await logging.setProjectId()
  await logging.setDetectedResource()

  // Create a LogSync transport, defaulting to `process.stdout`
  const log = logging.logSync(logname);
```
Given a fact that auth is used to generate resource based on environment, we might need to add a feature to support this through parameters, but I must admit I will wait to see if thats is going to be required by users. 
I opened an issue a while ago which should address this issue: [637](https://github.com/googleapis/nodejs-logging-winston/issues/637)